src/context/withBehandlingContext.jsx

```diff
@@ -4,15 +4,6 @@ import BehandlingerContext from './BehandlingerContext';
 export const withBehandlingContext = Component => {
     return props => {
         const behandlingerCtx = useContext(BehandlingerContext);
-
-        if (
```

Comment:
Jeg mener det skal være "trygt" å fjerne denne blokken, da all bruk av `withBehandlingContext` ligger under `MainContentWrapper`.

Reply:
Den er fin å ha siden den hindrer komponentene under å rendres hvis behandlinger er undefined. Den vil jo som du sier aldri være det når komponenten som trenger den ligger under `MainContentWrapper`, men da vil poenget til `withBehandlingContext` som en guard for dette bli litt borte. Om sjekken skal fjernes mener jeg man like gjerne kan fjerne hele `withBehandlingContext` og så legge sjekken inn i `MainContentWrapper`.

Reply:
Riktig.. Jeg så for meg at denne komponenten kommer til å fiske ut riktig sak (kontra hardkodet å velge første sak som nå). Men jeg er litt usikker på hvordan den saksvelgeren vil bli implementert.

Reply:
Skjønner. Like greit å la komponenten leve inntil videre da 👍